src/bitmessageqt/__init__.py

```diff
@@ -4004,6 +4015,11 @@ def on_context_menuInbox(self, point):
                     _translate("MainWindow", "Copy address to clipboard"),
                     self.on_action_ClipboardMessagelist)
                 self.popMenuInbox.addAction(self.actionClipboardMessagelist)
+                self._contact_selected = accountClass(
+                    tableWidget.item(currentRow, 1).data(QtCore.Qt.UserRole))
```

Comment:
You can probably suppress the warning, this looks like pylint doesn't understand we check that it's not boolean before.